src/rtpghi~.c

```diff
@@ -1,14 +1,30 @@
-//
-//  test.c
-//
-//
 //  Created by maximilian helligrath on 28.07.23.
 //
 
 #include <m_pd.h>
 #include <phaseret.h>
 #include <ltfat.h>
 
+#if !defined(PD_FLOATSIZE)
+  /* normally, our floats (t_float, t_sample,...) are 32bit */
+# define PD_FLOATSIZE 32
+#endif
+
+#if PD_FLOATSIZE == 32
+#define phaseret_rtpghi_init phaseret_rtpghi_init_s
+#define phaseret_rtpghi_done phaseret_rtpghi_done_s
+#define phaseret_rtpghi_set_tol phaseret_rtpghi_set_tol_s
+#define phaseret_rtpghi_set_causal phaseret_rtpghi_set_causal_s
+#define phaseret_rtpghi_execute phaseret_rtpghi_execute_s
+
```

Comment:
yes, something like this.
afair, you probably also want to define `LTFAT_DOUBLE` vs `LTFAT_SINGLE`  *before* including `phaseret.h`.
and then just do:
```C
#define rtpghi_init PHASERET_NAME(rtpghi_init)
// ...
```

and you just se `rtpghi_init()` instead of `phaseret_rtpghi_init_s()`.

the advantage over your solution is that you only need to `#define` once and both floatsizes are handled automatically (less code-duplication)